pkg/machine/applehv/config.go

```diff
@@ -30,11 +28,13 @@ type MMHardwareConfig struct {
 	Memory   int32
 }
 
-func (v Virtualization) Artifact() machine.Artifact {
-	return machine.Metal
+func GetVirtualizationProvider() machine.VirtProvider {
```

Comment:
Remove `Get` if possible

aplhv.VirtualizationProvider()

Should be enough

Reply:
fixed